cfme/tests/cli/test_appliance_console.py

```diff
@@ -374,3 +374,57 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
                 logger.info("{}: no result".format(rule))
         else:
             logger.info("{}: rule not found".format(rule))
+
+
+def test_appliance_console_dhcp(unconfigured_appliance, soft_assert):
+    """'ap' launches appliance_console, '' clears info screen, '1' configure network,
+    '1' configure DHCP, 'y' confirm IPv4 configuration, 'y' IPv6 configuration."""
```

Comment:
Please make the doc string more readable 
Example : 
   """ Command set - 
        1. 'ap' launches appliance_console, 
        2. ' ' clears info screen,
        3. '1' configure network,
        4. '1' configure DHCP, 
        5. 'y' confirm IPv4 configuration, 
        6. 'y' IPv6 configuration.
   """

Reply:
I like your suggestion better, but I followed the style of the rest of the file.